packages/discord.js/src/managers/GuildMemberManager.js

```diff
@@ -444,6 +444,24 @@ class GuildMemberManager extends CachedManager {
     return pruned;
   }
 
+  /**
+   * Disables a member's communication for a specified amount of time.
+   * <info>The user must be a member of the guild</info>
+   * @param {UserResolvable} user The user to timeout
```

Comment:
`GuildMemberManager#edit()` (which we'll be using) takes a `UserResolvable` as param, so no need to change that to a `GuildMemberResolvable`.

https://github.com/discordjs/discord.js/blob/852fae557e565a04bb65a8cf390d35e9faac10b2/packages/discord.js/src/managers/GuildMemberManager.js#L337-L343

Reply:
well it should be named guild member resolvable as we need member for that

Reply:
I don't see why though, as a `UserResolvable` also takes a `GuildMemberResolvable`. Thus making the range of possible parameters bigger. And yes, you are editing a `GuildMember`—But since you're in the scope of a `Guild` already (`GuildMemberManager`) it doesn't matter. And, `GuildMemberManager#ban()`, `#kick()` and `#edit()` all take `UserResolvable`s. So it's also better for consistency.

Reply:
it's just naming issue
it's self-explanatory that guild member must be in guild
in case of user probably not